### DISCUSSION TITLE
Problem with user query

### DISCUSSION THREAD

**Discussion by fabiosferreira** [community/community#24902] [Apps, API and Webhooks]
<p>I’d like to retrieve users data from the the GraphQL API v4 but I have found problems in some cases. For example, the query bellow returns the message  <strong>Could not resolve to a User with the login of ‘ReactiveX’</strong> :</p>
<pre><code>{
  user (login:"ReactiveX"){
            name
  }
}
</code></pre>
<p>If I use the REST API v3, the data are retrieved normally.</p>
<p><a href="https://api.github.com/users/ReactiveX" rel="nofollow noopener">https://api.github.com/users/ReactiveX</a></p>

**Comment by lee-dohm** [comment, marked as answer]
<p>You’re running into this problem because <a href="https://github.com/ReactiveX" rel="nofollow noopener">ReactiveX</a> is an organization, not a user. The following query works:</p>
<pre><code>{
  organization(login: "ReactiveX") {
    name
  }
}
</code></pre>

**Comment by fabiosferreira** [comment]
<p>Thank you very much <a class="mention" href="/lee-dohm">@lee-dohm</a>!</p>
